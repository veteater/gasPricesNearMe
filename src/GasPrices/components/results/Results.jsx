import React from 'react';
import Station from '../station/Station';

const Results = () => {
  return (
    <article className="results">
      <Station />
    </article>
  );
};

export default Results;