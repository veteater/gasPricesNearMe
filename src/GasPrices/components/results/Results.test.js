import React from 'react';
import { shallow } from 'enzyme';
import Results from './Results';

describe('Results', () => {
  it('renders without crashing', () => {
    const component = shallow(<Results />);
    expect(component).toMatchSnapshot();
  });
});
