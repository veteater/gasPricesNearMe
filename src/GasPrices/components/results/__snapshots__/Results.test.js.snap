// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results renders without crashing 1`] = `
<article
  className="results"
>
  <Station />
</article>
`;
